src/connect.js

```diff
@@ -19,8 +19,15 @@
  * @param {Object} baseParams
  * @return {Function}
  */
-function connect(params) {
-  // write code here
+function connect(params = 0) {
+  const rezultObj = { ...params };
+  return function addParams(args = rezultObj) {
```

Comment:
Good, but you can write simply 
```suggestion
  return (extraParams) => ({ ...params, ... extraParams });
```